### ISSUE TITLE
Camera Images Overlap

### ISSUE THREAD

**Issue by os1a** [waymo-research/waymo-open-dataset#120]
Hi,

I am trying to write a code to construct a panoramic image from the three front images (front, front_left, front_right). 
Could you please specify how much is the overlap between the images? 
Is it the same for all images in the dataset?

**Comment by peisun1115** [comment]
The paper (https://arxiv.org/pdf/1912.04838.pdf) talks about the HFOV. 

It is +/- 25.2 degree. We have 8 cameras in total (5 released). You can then compute the overlap

**Comment by os1a** [comment]
Thanks for your answer.

Is the overlap the same for all recorded sequences? I have tried to write a simple code to concatenate the three front images and here is the result of the same code running on two sequences:

![0100_result](https://user-images.githubusercontent.com/10829734/78453938-2b7c6a00-7695-11ea-82b3-3ec35c427476.png)

![0040_result](https://user-images.githubusercontent.com/10829734/78453946-3a631c80-7695-11ea-9792-5766d19f8384.png)

What I can see is that the height of the camera is different for these two scenarios?

**Comment by peisun1115** [comment]
Yes, they are the same. I have never concatenated them this way on my own. @henrikkretzschmar has done it. Maybe he can give some insights.

**Comment by henrikkretzschmar** [comment]
Thanks for reaching out!

The overlap is the same for all scenes in the dataset. Concatenating the images taken by the different cameras will not result in a perfect panorama image without any overlap. We found that such simple panorama images work really well for visualization, though.

Note that the dimensions of the images taken by the side cameras are different from the other cameras. If you want to include those in your panorama images as well, you could concatenate them by padding the top part of the side images.

**Comment by jakubLangr** [comment]
Hi @os1a did you ever get this working? Would love to see some panorama images!

**Comment by os1a** [comment]
@jakubLangr I found out that the simple concatenation of the images by padding the side images is not perfect but generally OK. You can see some examples above in my question. Of course it would be nice to find a solution to make it better.
